### PR TITLE
Faradawn [Word Display Page]

### PR DESCRIPTION
**Pull request by faradawn**
Made a Word Display Page. I think the Example Sentence part is quite complete? If there is anything I need to add or redo, just let me know!

### CODE REVIEW

**Review by Miclin1024 — COMMENTED** [review]
This looks great! The implementation for example sentence is beautiful. Just a few small things before we can merge this: 
- The alignment in the word definition is a bit off, the definition itself always goes to the second line despite there's ample space behind the part of speech.
- Use serif fonts for example sentences. We have a `use-serif` CSS class for that.
- You should always indent using 4 spaces. (You used 2 in some part of your code)

File: static/styles/learning.css [hunk 1]
```diff
@@ -83,4 +84,22 @@ ul.definition-list {
     ul.definition-list {
         padding: 0;
     }
-}
\ No newline at end of file
+
+}
+
+/* Word Displace - Example Sentences */
+.ExampleCard{
```
Comment (Miclin1024) [inline static/styles/learning.css]: Remember to add a space before opening curly braces.

File: learning/src/components/ExampleSentences.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import React from 'react'
```
Comment (Miclin1024) [inline learning/src/components/ExampleSentences.js]: Also another thing is to make sure that you add semicolons at the end of all javascript expressions. I use eslint to catch styling errors like this (as well as the indentions). An eslintrc has already been set up with the conventions that we are using. All you need to do is to install the extension in your IDE, or run the command if you prefer that.

**Comment by faradawn** [comment]
Hi, Thanks for the feedback! I will fix the alignment issue, serif font,
and “unprofessional” syntax. Thank a lot for the comments. Will start
refining the code as soon as possible : )

Faradawn

**Comment by chenyx512** [comment]
close as already merged in previous PR

**Pull request closed without merging**